e2e/README.md

```diff
@@ -15,19 +15,67 @@ for running all test. It by default uses the `StatusChain` network.
 
 #### Usage
 
+First of all you need to export an ACCOUNT_PASSWORD environment variable. It should be a passphrase
+that was used to generate accounts used in tests. If you don't know this variable for default accounts
+you will have to create your own accounts and request some funds from rinkeby or ropsten faucet.
+Please see Preparation section for details.
+
 To use the `ropsten` network for testing using network name:
 
 ```bash
-go test -v ./e2e/... -network=ropsten
+ACCOUNT_PASSWORD=test go test -v ./e2e/... -p= 1 -network=ropsten
 ```
 
 To use the `rinkeby` network with chain id `4` for testing:
 
 ```bash
-go test -v ./e2e/... -network=4
+ACCOUNT_PASSWORD=test go test -v ./e2e/... -p=1 -network=4
 ```
 
+#### Preparation
```

Comment:
For tests we keep already pregenerated test-accounts files in the source tree, right? These instructions describe how to make/update them again from scratch.
Perhaps that should be more emphasized in the doc because "Preparation" title can confuse people into thinking that you should do this before running tests.